### PR TITLE
fix: drop isolate, modules & bus commands

### PR DESCRIPTION
**Pull request by lili2311**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?
- `snyk modules` has been used only 4 times in two weeks and is an undocumented legacy
command to just show deps in `node_modules`, we have a better way to see this now
with `--print-deps` for any project type

- `isolate` command is orphaned, untested and not called by anything else

### CODE REVIEW

File: src/cli/args.ts [hunk 1]
```diff
@@ -153,15 +153,9 @@ export function args(rawArgv: string[]): Args {
...
-      'protect',
-      'test',
-      'modules',
-      'monitor',
-      'wizard',
-      'ignore',
-      'woof',
-    ].indexOf(command) !== -1
+    ['protect', 'test', 'monitor', 'wizard', 'ignore', 'woof'].indexOf(
+      command,
```
Comment (aviadhahami) [inline src/cli/args.ts]: maybe [Array.includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)?

Comment (dkontorovskyy) [reply src/cli/args.ts]: We still support Node 6. Array.includes is poorly supported there, if not mistaken

Comment (lili2311) [reply src/cli/args.ts]: Let's raise a separate PR to stop testing on 6

**Comment by lili2311** [comment]
https://github.com/snyk/snyk/issues/788

**Pull request merged**